components/webhook-mgt/org.wso2.carbon.identity.webhook.management/src/main/resources/dbscripts/h2.sql

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE IF NOT EXISTS IDN_WEBHOOK (
+    ID INTEGER NOT NULL AUTO_INCREMENT,
+    UUID VARCHAR(36) NOT NULL,
+    ENDPOINT VARCHAR(2048) NOT NULL,
+    DESCRIPTION VARCHAR(1024),
+    SECRET VARCHAR(1024) NOT NULL,
+    VERSION VARCHAR(32) NOT NULL,
+    EVENT_SCHEMA_NAME VARCHAR(64) NOT NULL,
+    EVENT_SCHEMA_URI VARCHAR(255) NOT NULL,
+    EVENT_SCHEMA_VERSION VARCHAR(32) NOT NULL,
+    STATUS VARCHAR(16) NOT NULL,
+    TENANT_ID INTEGER NOT NULL,
+    CREATED_AT TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    UPDATED_AT TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    PRIMARY KEY (ID),
+    UNIQUE(UUID),
+    UNIQUE(TENANT_ID, ENDPOINT)
+    );
+
+-- Table: IDN_WEBHOOK_EVENTS
+CREATE TABLE IF NOT EXISTS IDN_WEBHOOK_EVENTS (
+    ID INTEGER NOT NULL AUTO_INCREMENT,
+    WEBHOOK_ID INTEGER NOT NULL,
+    EVENT_NAME VARCHAR(255) NOT NULL,
+    UNIQUE (WEBHOOK_ID, EVENT_NAME),
+    PRIMARY KEY (ID),
+    UNIQUE(WEBHOOK_ID, EVENT_NAME),
```

Comment:
UNIQUE (WEBHOOK_ID, EVENT_NAME), is duplicated

Reply:
fixed